src/applications/simple-forms/form-upload/components/CustomReviewTopContent.jsx

```diff
@@ -40,7 +40,7 @@ const CustomReviewTopContent = () => {
           <div className="vads-u-display--flex vads-u-flex-direction--column">
             <span
               className="vads-u-font-weight--bold"
-              style={{ 'word-break': 'break-all' }}
+              style={{ wordBreak: 'break-all' }}
```

Comment:
This is not really related to the rest of the PR but it was raising a Warning...

Reply:
I would revert this and ignore the warning,  `word-break` is a CSS, not javascript property. camelCasing it will  only make this CSS style be ignored.

Reply:
Oh thanks for catching that! Fixed!